server/app/models.py

```diff
@@ -11,116 +11,21 @@ def get_current_utc():
-
-    def __repr__(self):
-        return f'<User {self.username}>'
-
-    def to_dict(self):
-        return {
-            'user_id': self.user_id,
-            'username': self.username,
-            'email': self.email,
-            'password_hash': self.password_hash,
-            'role': self.role,
-            'is_active': self.is_active,
-            'confirmed_admin': self.confirmed_admin
-        }
-
-
-class Invitation(db.Model):
-    __tablename__ = 'invitations'
-    Invitation_id = db.Column(db.Integer, primary_key=True)
-    token = db.Column(db.String(50), unique=True, nullable=False)
-    email = db.Column(db.String(50), nullable=False)
-    created_at = db.Column(db.DateTime, default=get_current_utc, nullable=False)
-    expiry_date = db.Column(db.DateTime, nullable=False)
-    is_used = db.Column(db.String(1), nullable=False, default='0')
-
-    # Foreign key relationship
-    user_id = db.Column(db.Integer, db.ForeignKey('users.user_id'), nullable=False)
-
-    # Set expiry date to 72 hours after date invitation was created : Winnie
-    def calculate_expiry_date(self):
-        # Calculate expiry date as 72 hours after created_at
-        self.expiry_date = self.created_at + timedelta(hours=72)
-
-    def __init__(self, token, email, expiry_date, user_id, is_used='0'):
-        self.token = token
-        self.email = email
-        self.expiry_date = expiry_date
-        self.user_id = user_id
-        self.is_used = is_used
-
-    def __repr__(self):
-        return f'<Invitation {self.Invitation_id}>'
-
-    def to_dict(self):
-        return {
-            'Invitation_id': self.Invitation_id,
-            'token': self.token,
-            'email': self.email,
-            'created_at': self.created_at,
-            'expiry_date': self.expiry_date,
-            'is_used': self.is_used,
-            'user_id': self.user_id
-        }
-
-
-# Models : Winnie
-    
-class Store(db.Model):
-    __tablename__ = 'stores'
-
-    store_id = db.Column(db.Integer, primary_key=True)
-    store_name = db.Column(db.String(50), nullable=False)
-    location = db.Column(db.String(50), nullable=False)
-
-    # Converting store object to dictionary
-    def to_dict(self):
-        return {
-            'store_id': self.store_id,
-            'store_name': self.store_name,
-            'location': self.location
-        }
-
-    products = relationship('Product', backref='store')
-    
-    # temporarily commented out until inventory table merged by other team member
-    # inventory = relationship('Inventory', backref='store')
-
-    def __repr__(self):
-        return f'Store(id={self.store_id}, name={self.store_name}, name={self.location})'
-
-class Product(db.Model):
-    __tablename__ = 'products'
-
-    product_id = db.Column(db.Integer, primary_key=True)
-    product_name = db.Column(db.String(50), nullable=False)
-    buying_price = db.Column(db.Integer, nullable=False)
-    selling_price = db.Column(db.Integer, nullable=False)
```

Comment:
Seems you have deleted these other tables. Is this the expected outcome or has it been done mistakenly?